src/Button.js

```diff
@@ -7,6 +7,8 @@ class Button extends React.Component {
         accent: PropTypes.bool,
         className: PropTypes.string,
         colored: PropTypes.bool,
+        component: PropTypes.any,
```

Comment:
Shouldn't this be `PropTypes.element` ? Or if we want to allow string as well (but not sure we want that), we could do a `oneOfType`...